varia/sass/blocks/heading/_config.scss

```diff
@@ -19,15 +19,6 @@ $config-heading: (
 			"h1": map-deep-get($config-global, "font", "size", "xxxxl"),
 		),
 		// Letter spacing
-		"line-height": (
```

Comment:
All themes use the heading `line-height` for every `h*` element, so I thought we could remove these mappings. If we wanted to keep these variables, it's still possible, although they should be provided by Global Styles.

Reply:
Interesting. Didn’t realize that but if they aren’t being used anywhere, happy to have these removed :-)